ui/perf.html

```diff
@@ -32,6 +32,7 @@
       <a ng-class="{active: $state.current.name.startsWith('compare')}" class="btn btn-view-nav" ui-sref="comparechooser">Compare</a>
       <a ng-class="{active: $state.current.name.indexOf('alerts') >= 0}" class="btn btn-view-nav" ui-sref="alerts({id: null, status: null, framework: null, filter: null, hideImprovements: null})">Alerts</a>
       <a ng-class="{active: $state.current.name.indexOf('e10s') >= 0}" class="btn btn-view-nav" ui-sref="e10s">e10s</a>
+      <a ng-class="{active: $state.current.name.indexOf('e10s_trend') >= 0}" class="btn btn-view-nav" ui-sref="e10s_trend">e10s-Trend</a>
```

Comment:
Let's dump these two into a "dashboard" tab, so as not to clutter up the top bar. You can model this after the help menu (just below), though you can just put the "ul" elements inline (since we won't be sharing the values with anything else).